content/docs/notebooks/setup.md

```diff
@@ -38,24 +38,7 @@ set up your Kubeflow deployment in your environment of choice (locally, on
 premises, or in the cloud).
 
 When Kubeflow is running, access the Kubeflow UI as described in the
-getting-started guide for your chosen environment. For example:
-
-* If you deployed Kubeflow on Google Cloud Platform (GCP), the Kubeflow UI is
-  available at the following URI:
-
-    ```
-    https://<deployment_name>.endpoints.<project>.cloud.goog/
-    ```
-
-* If you set up port forwarding to the Ambassador service, the Kubeflow UI is
-  available at the following URI:
-
-    ```
-    http://localhost:8080/
-    ```
-
-* For other environments, see the getting-started guide for your chosen 
-  environment.
+getting-started guide for your chosen environment. Follow [accessing Kubeflow UIs guide](https://www.kubeflow.org/docs/other-guides/accessing-uis/) for instructions on how to connect to Kubeflow UI.
```

Comment:
Thanks for this update! Just one comment: When adding a link to a page within the doc site, it's best to use a relative link like this:

```
[accessing Kubeflow UIs guide](/docs/other-guides/accessing-uis/)
```

We use relative links for two reasons. Firstly, when people are looking at a preview version of the site, a relative link keeps them within the preview whereas an abolute link jumps out to the production server. Secondly, when we have translated docs in other languages, which we hope to do sometime soon, the relative link works with the internationalization tools.

Reply:
Thanks for the info on relative links. changes is done.